Updated docs with enterprise update details (#6498)
MicrosoftDocs/visualstudio-docs: docs/install/update-a-network-installation-of-visual-studio.md

::: moniker-end

> * Administrators in organizations which use enterprise tools like Windows Server Update Services and System Center Configuration Manager can update client deployments of Visual Studio with [Updates using Microsoft Update Catalog](http://TODOLink). These updates are designed to work with client deployments of Visual Studio which were created from a network layout. 

Reviewer: ```suggestion
* Administrators in organizations that use enterprise tools such as Windows Server Update Services and System Center Configuration Manager can update client deployments of Visual Studio with [Updates using Microsoft Update Catalog](http://TODOLink). These updates are designed to work with client deployments of Visual Studio that were created from a network layout. 
```
